tests/iles.ts

```diff
@@ -5,6 +5,9 @@ export async function test(options: RunOptions) {
 	await runInRepo({
 		...options,
 		repo: 'ElMassimo/iles',
+		overrides:{
+			"@vitejs/plugin-vue": true
+		},
```

Comment:
Why is this needed? Is it because the dependency isn't at the top-level package.json?

Reply:
yes, we only scan the top level package.json. in monorepos you have to tell us you want/need it

Reply:
Gotcha